README.md

```diff
@@ -65,6 +65,16 @@ $ cp themes/hugo_theme_pickles/exampleSite/config.toml .
 
 Now, you can start editing this file and add your own information!
 
+## Customisation
+
+To hide the date in any page or post, set the `hideDate` parameter in the front-matter:
```

Comment:
Looks like Hugo  spells `front matter` without the hyphen on its site.

Reply:
Thanks, fixed!